packages/calcite-components/src/components/tile-group/tile-group.scss

```diff
@@ -0,0 +1,36 @@
+:host {
+  box-sizing: border-box;
+  display: inline-block;
+
+  .tile-group {
+    display: grid;
+    grid-auto-rows: minmax(auto, 1fr);
+    grid-template-columns: repeat(auto-fit, minmax(140px, 1fr));
```

Comment:
nitpick: move this rule into `:host([scale="m"]) { .tile-group {` for clarity

Reply:
@jcfranco  what do you think about Ali's idea here?  I did this mainly because in the past if there was a default case in some javascript logic, setting things up like this minimized code branches, but maybe we want to do things differently with CSS?

Reply:
I agree and think we should consider updating our approach to lean more on being explicit vs having implicit defaults styles.